corehq/apps/events/models.py

```diff
@@ -248,6 +277,26 @@ def delete(self, using=None, keep_parents=False):
     def status(self):
         return self.attendee_list_status
 
+    def get_total_attendance_takers(self):
+        return len(self.attendance_taker_ids)
+
+    @property
+    def case_id(self):
```

Comment:
If this is fake, it probably should be named as such.

Reply:
That is a good point, I have renamed it to `fake_case_id`. :+1: